test/paymentsSpec.js

```diff
@@ -79,6 +79,13 @@ describe('Payments', function () {
 
     this.mockPlugin1 = this.ledgers.getPlugin('mock.test1.')
     this.mockPlugin2 = this.ledgers.getPlugin('mock.test2.')
+
+    // Populate the curve cache.
```

Comment:
Isn't this a local route? So it should already have a curve when it's added to the routing table?